moveit_ros/moveit_servo/src/servo_node.cpp

```diff
@@ -321,7 +323,8 @@ void ServoNode::servoLoop()
       RCLCPP_WARN_STREAM(LOGGER, "Command type has not been set, cannot accept input");
     }
 
-    if (next_joint_state && (servo_->getStatus() != StatusCode::INVALID))
+    if (next_joint_state && (servo_->getStatus() != StatusCode::INVALID) &&
+        (servo_->getStatus() != StatusCode::HALT_FOR_COLLISION))
```

Comment:
fun bug: Without this change, the robot would be commanded to go to zero positions when it halted for collision :)